### ISSUE TITLE
Parameter lists hard to parse by eye

### ISSUE THREAD

**Issue by dstansby** [napari/napari-sphinx-theme#111]
Parameter lists in `napari` API docs are hard to parse by eye for me - the black wall of text isn't very good for reading:
![Screenshot 2022-06-27 at 13 02 57](https://user-images.githubusercontent.com/6197628/175936705-8f4d4cfa-9813-4d2b-afb2-32c2e5841f3a.png)

It would be good if there could be some distinction between parameter names and values, perhaps using colour? Or putting each parameter on a newline instead of one continuous line?

**Comment by jni** [comment]
I prefer lines rather than colour. I find the alternating colour of some syntax highlighting very psychedelic and equally hard to read.

Ultimately I think part of the problem is our massive signatures rather than the theme. We can probably group a bunch of them to make this simpler. For example all of the size, face_color etc will go into "style=", all of scale, translate etc into "transforms=", properties (deprecated) will go away... Not discussed but maybe name/metadata, opacity, blending can go into "layer_options=". Then you end up with something a lot more reasonable and readable.

**Comment by Czaki** [comment]
Agree, but this refactor should not be only in the constructor but also on the inner Layers structure. They have too many attributes, and they should be grouped.

**Comment by psobolewskiPhD** [comment]
I think this is much improved with #134 
![image](https://github.com/napari/napari-sphinx-theme/assets/76622105/2efc59fd-82cc-4f86-8f03-15fdbf313f3b)

What do y'all think?

**Comment by jni** [comment]
I think we need to simplify our parameter lists 😂 

But yes it's so much better 😭 ❤️
